rules-reviewed/technology-usage/configuration-management.windup.xml

```diff
@@ -27,5 +27,69 @@
                 <technology-tag level="INFORMATIONAL">Spring Cloud Config</technology-tag>
             </perform>
         </rule>
+        <rule id="configuration-management-0200">
+            <when>
+                <file filename="application{profile}.{extension}"   />
+            </when>
+            <perform>
+                <classification title="Spring application properties detected" category-id="information" effort="0">
+                    <description>Spring application properties detected</description>
+                </classification>
+                <technology-tag level="INFORMATIONAL">Spring Properties</technology-tag>
+            </perform>
+            <where param="profile">
+                <matches pattern=".+" />
```

Comment:
The dot-plus means 'any char one or more times' which means there has to be something there in the file name. If you change it to `<matches pattern=".*"/>` it means zero or more times so it accepts nothing between `application` and `.properties`
```suggestion
                <matches pattern=".*" />
```